docs/user-guide/writing-your-docs.md

```diff
@@ -365,11 +374,14 @@ specific page. The following keys are supported:
     1. A title defined in the [nav] configuration setting for a document.
     2. A title defined in the `title` meta-data key of a document.
     3. A level 1 Markdown header on the first line of the document body.
+       Please note that [Setext-style] headers are not supported.
     4. The filename of a document.
 
     Upon finding a title for a page, MkDoc does not continue checking any
     additional sources in the above list.
 
+[Setext-style]: https://github.com/mkdocs/mkdocs/issues/1826#issuecomment-511078194
```

Comment:
Let's link to the actual definition of Setext-style headers: https://daringfireball.net/projects/markdown/syntax#header